src/utils/StringOp.hh

```diff
@@ -21,6 +21,11 @@
 
 namespace StringOp
 {
+	/** Convert string to an enum type 'T' as a numeric value.
+	  */
+	template<typename T, typename std::enable_if<std::is_enum<T>::value>>
+	[[nodiscard]] std::optional<T> stringTo(std::string_view s);
+
```

Comment:
See my remark in `saveShortcuts`. I don't think it's ever useful to save/parse an enum value as the numerical representation of that enum. So this function should be removed.